introduction.md

```diff
@@ -3,3 +3,7 @@ Introduction
 
 Hi, I'm entantoencuanto and I'm learning things to master GitHub
 I'm especially interested in Ruby and JavaScript paths, but I find interesting the others paths
+Deadly Skills:
+* git
+* Ruby
+* JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?